zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -5773,6 +5773,12 @@ components:
         name:
           description: Name of this process definition.
           type: string
+        isLatestVersion:
+          description: >
+            Whether to only return the latest version of each process definition.
+            When using this filter, pagination functionality is limited on clusters running on Elasticsearch or OpenSearch.
+            The response contains no `startCursor` in the `page`, requests ignore the `from` and `before` in the `page`.
```

🔧 Like we discussed offline, let's make clear what works.
```suggestion
            Whether to only return the latest version of each process definition.
            When using this filter, pagination functionality is limited; you can only paginate forward using `after` and `limit`.
            The response contains no `startCursor` in the `page`, and requests ignore the `from` and `before` in the `page`.
```